hnn_core/gui/_viz_manager.py

```diff
@@ -1026,3 +1030,10 @@ def _simulate_edit_figure(self, fig_name, ax_name, simulation_name,
             buttons.children[0].click()
         elif operation == "clear":
             buttons.children[1].click()
+
+
+def _is_simulation(data):
+    """Determines if saved data is a simulation."""
+    if data['net'] is not None:
+        return True
+    return False
```

Comment:
```suggestion
    return data['net'] is not None
```
Great work! Just a few minor refactoring suggestions. Feel free to accept them if you agree.

Reply:
Done